src/obb/server/controllers/index.cljc

```diff
@@ -0,0 +1,9 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.controllers.index
+  "Controller to handle the root/index request to the server")
+
+(defn handler
+  "Returns a intro to the game"
+  [request]
+  {:body "Hello World!"
+   :status 200})
```

Comment:
Sure, go ahead. I also agree with your argument about dropping HTTP from the name.